packages/ecs-gpu-init/ecs-gpu-init.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Initialize ECS GPU config
+# This has to be executed after the kernel modules are loaded
+# otherwise the userspace component of the driver will fail to
+# query the /dev devices
+After=load-kernel-modules.service
```

Comment:
```suggestion
# This has to be executed after the kernel modules are loaded
# otherwise the userspace component of the driver will fail to
# query the /dev devices
After=load-kernel-modules.service
Requires=load-kernel-modules.service
```